src/lib/badge/badge.scss

```diff
@@ -0,0 +1,64 @@
+$mat-badge-size: 22px !default;
+
+.mat-badge {
+  position: relative;
+  display: inline-block;
+
+  &.mat-badge-above {
```

Comment:
This nesting seems like it's bumping the specificity unnecessarily. Consider moving it out instead?

Reply:
I changed this quite a bit, you might want to re-review.